lookout/style/format/benchmarks/general_report.py

```diff
@@ -257,13 +256,13 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
             filepath = fix.head_file.path
             if fix.error:
                 continue
-            if self.config["aggregate"]:
+            if self.config["common"]["aggregate"]:
```

Comment:
Why common?

Reply:
because it is language independent. The same as here: https://github.com/src-d/style-analyzer/pull/650#discussion_r